DXMainClient/DXGUI/Multiplayer/GameLobby/GameLobbyBase.cs

```diff
@@ -687,16 +689,16 @@ private void DeleteSelectedMap(XNAMessageBox messageBox)
         {
             try
             {
-                Logger.Log("Deleting map " + Map.BaseFilePath);
-                File.Delete(Map.CompleteFilePath);
-                foreach (GameMode gameMode in GameModeMaps.GameModes)
-                {
-                    gameMode.Maps.Remove(Map);
-                }
+                MapLoader.DeleteCustomMap(GameModeMap);
 
                 tbMapSearch.Text = string.Empty;
                 if (GameMode.Maps.Count == 0)
+                    // this will trigger another GameMode to be selected
                     GameModeMap = GameModeMaps.Find(gm => gm.GameMode.Maps.Count > 0);
```

Comment:
From the style guide:

> Braceless code block bodies should be made only when both code block head and body are single line, statements split into multiple lines and nested braceless blocks are not allowed within braceless blocks

While it's technically still a single line, the comment here makes it so it's not visually a single line. So add braces around the code block here.